scripts/test-reviewer.mjs

```diff
@@ -1,212 +1,259 @@
+const reviewIndexTemplateFilePath = path.resolve(scriptsDirectory, 'review-index-template.mustache');
+
+// create directories if not exists
+fs.existsSync(reviewBuildDirectory) || fs.mkdirSync(reviewBuildDirectory);
 
-const testDir = path.resolve('.', 'tests');
-const templateFile = path.resolve('.', 'scripts', 'review-template.mustache');
-const templateIndexFile = path.resolve('.', 'scripts', 'review-index-template.mustache');
-const reviewDir = path.resolve('.', 'review');
 const allTestsForPattern = {};
-const support = JSON.parse(fse.readFileSync(path.join(testDir, 'support.json')));
+const support = JSON.parse(fse.readFileSync(supportFilePath));
+
 let allATKeys = [];
 support.ats.forEach(at => {
-	allATKeys.push(at.key);
+    allATKeys.push(at.key);
 });
+
 const scripts = [];
 
-const getPriorityString = function(priority) {
-  priority = parseInt(priority);
-  if (priority === 1) {
-    return 'required';
-  }
-  else if (priority === 2) {
-    return 'optional';
-  }
-  return '';
+const getPriorityString = function (priority) {
+    priority = parseInt(priority);
+    if (priority === 1) {
+        return 'required';
+    } else if (priority === 2) {
+        return 'optional';
+    }
+    return '';
 }
 
-fse.readdirSync(testDir).forEach(function (subDir) {
-  const subDirFullPath = path.join(testDir, subDir);
-  const stat = fse.statSync(subDirFullPath);
-  if (
-    stat.isDirectory() &&
-    subDir !== 'resources'
-  ) {
-
-    // Initialize the commands API
-    const commandsJSONFile = path.join(subDirFullPath, 'commands.json');
-    const commands = JSON.parse(fse.readFileSync(commandsJSONFile));
-    const commAPI = new commandsAPI(commands, support);
-
-    const tests = [];
-
-    const referencesCsv = fs.readFileSync(path.join(subDirFullPath, 'data', 'references.csv'), 'UTF-8');
-    const reference = referencesCsv.split(/\r?\n/).find(s => s.startsWith('reference,')).split(',')[1];
-
-    const scriptsPath = path.join(subDirFullPath, 'data', 'js');
-    fse.readdirSync(scriptsPath).forEach(function (scriptFile) {
-      let script = '';
-      try {
-        const data = fs.readFileSync(path.join(scriptsPath, scriptFile), 'UTF-8');
-        const lines = data.split(/\r?\n/);
-        lines.forEach((line) => {
-          if (line.trim().length)
-            script += '\t' + line.trim() + '\n';
+fse.readdirSync(testsDirectory).forEach(function (directory) {
+    const testPlanDirectory = path.join(testsDirectory, directory);
+    const testPlanBuildDirectory = path.join(testsBuildDirectory, directory);
+    const stat = fse.statSync(testPlanDirectory);
+
+    if (stat.isDirectory() && directory !== 'resources') {
+        // Initialize the commands API
+        const commandsJSONFile = path.join(testPlanBuildDirectory, 'commands.json');
+        const commands = JSON.parse(fse.readFileSync(commandsJSONFile));
+        const commAPI = new commandsAPI(commands, support);
```

Comment:
In the spirit of writing the scripts in a more standard style (which I appreciate) maybe we could do `import { commandsAPI as CommandsAPI }` and rename `commAPI` `commandsAPI`?